plants-backend-adapter-service/src/test/java/uk/gov/defra/plants/backend/component/TradeAPIDoATest.java

```diff
@@ -0,0 +1,127 @@
+package uk.gov.defra.plants.backend.component;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.urlPathMatching;
+import static org.apache.http.HttpStatus.SC_OK;
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.defra.plants.common.constants.OrganisationDetails.INDIVIDUAL_USER_WITH_ONE_AGENT;
+import static uk.gov.defra.plants.commontest.security.session.Users.givenDOAExporter;
+
+import com.github.tomakehurst.wiremock.client.WireMock;
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import uk.gov.defra.plants.backend.component.helpers.ComponentTestFramework;
+import uk.gov.defra.plants.backend.representation.organisation.domain.DoaContactOrganisations;
+import uk.gov.defra.plants.backend.representation.organisation.domain.DoaOrganisationsResponse;
+import uk.gov.defra.plants.backend.representation.organisation.tradeapi.ApplicationPermissionRequestPermissionModel;
+import uk.gov.defra.plants.backend.representation.organisation.tradeapi.OrganisationPermissionListingResponse;
+import uk.gov.defra.plants.backend.representation.organisation.tradeapi.OrganisationPermissionOrgListing;
+import uk.gov.defra.plants.backend.representation.organisation.tradeapi.OrganisationPermissionOrgResponse;
+import uk.gov.defra.plants.common.json.ItemsMapper;
+
+public class TradeAPIDoATest {
+
+  private static final String PERMISSION_URI = "/trade-delegated-authority/v1/Permission/";
+  private static final String DOA_ORGANISATIONS_RESOURCE_NAME = "/Permissions/Organisations";
+  public static final String CONTACT_ID = "f72591a1-6d8b-e911-a96f-000d3a29b5de";
+  public static final String SERVICE_ID = "8b5214ee-62b6-e811-a954-000d3a29b5de";
```

Comment:
should this id be in here?

Reply:
tricky one that....we need an id in the test to match against the test config used by the server that is started when running the test. As the test config has been removed, this test wouldn't work anyway.  
Ideally we would refactor all the tests to use values from config files, instead of literal values like this, which I think is the next step.

I also checked this service_id value across the environments and it is only used for stubbing.

Reply:
ok